### PR TITLE
refactor: ScyllaDB/Elasticsearch Query & Insertion script

### PR DESCRIPTION
**Pull request by vaishnav-mk**
Removed deprecated practices:
- Eliminated references to `doc_type` in Elasticsearch queries (no longer required).
- Mentioned default port number for Elasticsearch
- Addressed a deprecation warning by using client.options for ignoring index creation existence (400 status).
- Enhanced Readability: Improved code formatting and removed unnecessary comments for better maintainability.

### CODE REVIEW

File: scylla_elasticsearch_integration/query_data_from_scylla+elastic.py [hunk 1]
```diff
@@ -4,94 +4,79 @@
...
-from cassandra import ConsistencyLevel
-from cassandra.concurrent import execute_concurrent_with_args
 
-
-## Script args and help
+## Script args and Help
 parser = argparse.ArgumentParser(add_help=True)
 
-parser.add_argument('-s', action="store", dest="SCYLLA_IP", default="127.0.0.1")
-parser.add_argument('-e', action="store", dest="ES_IP", default="127.0.0.1")
```
Comment (DanielHe4rt) [inline scylla_elasticsearch_integration/query_data_from_scylla+elastic.py:20]: This is related to any docker base configuration?

**Review by DanielHe4rt — COMMENTED** [review]
Thanks for the contribution! I just have one question about it:

File: scylla_elasticsearch_integration/query_data_from_scylla+elastic.py (hunk 1 above)
Comment (vaishnav-mk) [reply scylla_elasticsearch_integration/query_data_from_scylla+elastic.py:20]: No, I'm running scylla and elastic locally without docker containers and elastic requires you to specify the host and the port number
image

Comment (DanielHe4rt) [reply scylla_elasticsearch_integration/query_data_from_scylla+elastic.py:20]: Seems good! I'll test it tomorrow if no one test it before and merge. 

Cheers for the contribution!

**Comment by DanielHe4rt** [comment]
LGTM!

File: scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py [hunk 2]
```diff
@@ -5,97 +5,91 @@
 ### Using elasticsearch-py ###
 import csv
 from cassandra.cluster import Cluster
+from cassandra import ConsistencyLevel
```
Comment (Daniel-Boll) [inline scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py:8]: I think mostly we can change this examples to use the scylla driver instead.

@DanielHe4rt wdyt?

File: scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py [hunk 3]
```diff
@@ -5,97 +5,91 @@
...
 def load_data(filename):
     data = []
     headers = []
-    with open(filename, "r") as f:
+    with open(filename, 'r') as f:
         reader = csv.reader(f)
         headers = next(reader) # read the headers line
-
-        for l in reader:
+
```
Comment (Daniel-Boll) [inline scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py:36]: There are multiple occurrences in the code of empty lines but with tabs, is this related to any python formatter?

Afaik not, but the I think it would be good to give a formatter a pass in the code anyways, not necessarily in this PR.

File: scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py [hunk 4]
```diff
@@ -5,97 +5,91 @@
...
-
+    
     print("")
-
-
+        
 if __name__ == "__main__":
-    headers, data = load_data("./catalog.csv")
-    insert_data(headers, data)
+    headers, data = load_data('./catalog.csv')
```
Comment (Daniel-Boll) [inline scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py:94]: Inconsistence between single and double quotes, which may be fixed with the formatter.

**Review by Daniel-Boll — APPROVED** [review]
LGTM 🙏

File: scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py (hunk 2 above)
Comment (DanielHe4rt) [reply scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py:8]: I first tested his sample with the scylla python driver and it worked perfectly.

**Comment by DanielHe4rt** [comment]
Hey @vaishnav-mk! Thanks for the contribution again.

I tested here and it work like a charm together with the ScyllaDB Python Driver. Also I did a couple of changes on the project's README that you can find already in the repository or in this commit https://github.com/scylladb/scylla-code-samples/commit/b80103cc11caf8d640edfcdc748b0b52e8ef970b.

I'll merge it right now!
